### PR TITLE
Add Caption component

### PR DESCRIPTION
**Pull request by ma12-co**
Screenshot from 2021-03-05 09-51-00

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/Caption/Caption.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div role="heading" class="app-navigation-caption">
```
Comment (skjnldsv) [inline src/components/Caption/Caption.vue:23]: I'm a bit confused by this. We already have a Caption component for the navigation.
Is this component navigation only? It's also missing documentation.

I think the name should be more straightforward, and maybe the component should be in a dedicated section of the docs? Like 'Text', where we could also put the highlight component as well? :thinking:

Comment (ma12-co) [reply src/components/Caption/Caption.vue:23]: > Is this component navigation only

It's used in both navigation and sidebar in Talk

> It's also missing documentation

Oopsss

> I think the name should be more straightforward, and maybe the component should be in a dedicated section of the docs? Like 'Text', where we could also put the highlight component as well? thinking

Not sure I get this, quick call?

**Comment by PVince81** [comment]
@ma12-co what's the difference with your other similarly titled PR ? https://github.com/nextcloud/nextcloud-vue/pull/1739

**Comment by ma12-co** [comment]
> @ma12-co what's the difference with your other similarly titled PR ? #1739

That component is for use within actions only.

File: src/components/Caption/Caption.vue (hunk 1 above)
Comment (ChristophWurst) [reply src/components/Caption/Caption.vue:23]: > Not sure I get this, quick call?

For transparency and posterity it would be cool to have the infos, decisions and so on here with the PR.

Comment (PVince81) [reply src/components/Caption/Caption.vue:23]: @ma12-co what was the results of the call ? Can you summarize ?

Comment (ma12-co) [reply src/components/Caption/Caption.vue:23]: Waiting for @jancborchardt 
Discussion here #1751

**Comment by ma12-co** [comment]
closing in favor of #1863
